config.toml

```diff
@@ -250,10 +378,10 @@ anchor = "smart"
     url = "https://themes.gohugo.io/"
 
     [[menu.global]]
-      name = "Showcase"
+      name = "作品展示"
       weight = 20
       identifier = "showcase"
-      url = "/showcase/"
+      url = "/zh/showcase/"
```

Comment:
@LinuxSuRen These you shouldn't be changing, otherwise the English version is gone...

Reply:
ok, Thank you